### PR TITLE
feat: add indicator r018

### LINKED ISSUE #90: indicators: add R018 Single bid received
Methodology:

Required OCDS fields: tender/procurementMethod, (tender/numberOfTenderers, OR tender/tenderers/id OR bids/details/tenderers/id)

Calculation method: For tenders  i  where tender/procurementMethod = 'open',

R018i=1  if  tender/numberOfTenderersi=1

### PR DESCRIPTION
**Pull request by yolile**
closes #90

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -37,5 +37,6 @@ repos:
           # https://github.com/seanmonstar/pretty-env-logger/pull/49
           # https://github.com/eminence/terminal-size/pull/49
           "-Aclippy::multiple-crate-versions",
+          "-Aclippy::doc-markdown",
```
Comment (yolile) [inline .pre-commit-config.yaml:40]: @jpmckinney I had to add this again so that I can commit. We can compare our rust/cargo versions this time.

Comment (jpmckinney) [reply .pre-commit-config.yaml:40]: Can you run `rustc --version` and `cargo --version` ?

Comment (yolile) [reply .pre-commit-config.yaml:40]: rustc 1.78.0-nightly (b6d2d841b 2024-03-05)
cargo 1.78.0-nightly (f772ec022 2024-03-01)

Comment (jpmckinney) [reply .pre-commit-config.yaml:40]: And `cargo clippy --version` (added later) ?

Comment (yolile) [reply .pre-commit-config.yaml:40]: clippy 0.1.78 (b6d2d84 2024-03-05)

Comment (jpmckinney) [reply .pre-commit-config.yaml:40]: Okay, I can reproduce now.

**Pull request merged**